ugs-core/src/resources/MessagesBundle_en_US.properties

```diff
@@ -32,9 +32,7 @@ mainWindow.helpKeyPlusMinus = In-/Descrease Steps\: KeyPad +/-
 mainWindow.helpKeyDivMul = Change Steps\: KeyPad DIVIDE/MULTIPLY
 mainWindow.helpKeyZero = Reset Zero\: KeyPad Insert; Numpad\: 0
 mainWindow.status.alarm = Alarm
-mainWindow.status.hold = Hold
-mainWindow.status.queue = Queue
-mainWindow.status.run = Run
```

Comment:
Pretty much all languages have these fields already, do they need to be removed?

Reply:
Sorry, I don't think I understand what you are saying. I'm not sure what your approach to managing translations is. Should I have not removed these?

Reply:
I think this ties in to another comment, I think I get it, put them back in, then map GRBL's status to these strings.

Reply:
Marking as done for tracking my tasks.